guide/payments/contacts.md

```diff
@@ -0,0 +1,229 @@
+    - file: sending-pick-contact
+      alt: Screen with a list of contacts to choose from
+      caption: Tapping "Link contact" presents the user with a contact list modal.
+    - file: pay-invoice-with-contact
+      alt: Invoice screen with a contact assigned
+      caption: A basic invoice with an assigned contact.
+    - file: pay-invoice-contact-with-tx
+      alt: Contact with a single transaction assigned
+      caption: A contact with an assigned transaction, but no re-usable addresses.
+    - file: pay-invoice-contact-with-tx-and-address
+      alt: Contact with a transaction and an address assigned
+      caption: A contact with an assigned transaction and a re-usable address.
+imagesHome:
+    - file: sending-home
+      alt: Home screen with amount input
+      caption: The user enters an amount and taps "Pay".
+    - file: sending-pick-contact
+      alt: Screen with a list of contacts to choose from
+      caption: Next, the user is asked to choose a contact to send to.
+    - file: sending-review
+      alt: Invoice screen with a contact assigned
+      caption: Final review of the payment before sending it.
+imagesReceive:
+    - file: receive-home
+      alt: Home screen with amount input
+      caption: The user enters an amount and taps "Request".
+    - file: receive-qr
+      alt: Payment request screen
+      caption: An invoice screen with minimal information.
+    - file: receive-qr-with-contact
+      alt: Payment request screen with a contact assigned
+      caption: The same invoice with a contact assigned.
+imagesActivity:
+    - file: activity
+      alt: Activity screen showing transactions with minimal information
+      caption: By default, payments contain little to no contextual information.
+    - file: activity-annotated
+      alt: Activity screen showing transactions with assigned contacts
+      caption: Assigning contacts to payments makes them much easier to understand and work with.
+    - file: activity-transaction
+      alt: Transaction details
+      caption: Transaction detail screens should have convenient options to assign a contact.
+    - file: sending-pick-contact
+      alt: Screen with a list of contacts to choose from
+      caption: The user picks a contact in a slide-up modal.
+    - file: activity-transaction-annotated
+      alt: Transaction details with assigned contact
+      caption: A payment with an assigned contact.
+
+---
+
+{% include picture.html
+    image = "/assets/images/guide/payments/contacts/contacts-header.jpg"
+    retina = "/assets/images/guide/payments/contacts/contacts-header@2x.jpg"
+    mobile = "/assets/images/guide/payments/contacts/contacts-header-mobile.jpg"
+    mobileRetina = "/assets/images/guide/payments/contacts/contacts-header-mobile@2x.jpg"
+    alt-text = "Illustration of large contact cards"
+    width = 1600
+    height = 400
+    layout = "full-width"
+%}
+
+
+# Contacts
+{:.no_toc}
+
+<!--
+
```

Comment:
Although I know the images are used from the UI kit it would still be handy to have the link to the Figma file

Reply:
Added.